conversion_examples/BlissMultiModal/make_xas.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
+
+import h5py
+import numpy
```

Comment:
[They](https://numpy.org/doc/2.2/user/absolute_beginners.html#how-to-import-numpy) recommend to import as `import numpy as np`

Reply:
```python
import numpy as np
import pandas as pd
...
```

I'm not a favor for the same reason I'm not in favor of single letter variable names.